lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -290,6 +317,38 @@ def assert_valid_attributes!(attributes)
           raise ArgumentError, "Provided argument contains invalid keys, valid keys are #{PolicyElement.column_names}"
         end
       end
+
+      # Returns a hash containing the ancestors of the root nodes, with the interstitial
+      # ancestor relationships represented by the key/value pairs.
+      def get_ancestor_id_tree(root_nodes)
+        Assignment.select_ancestor_ids(root_nodes).each_with_object({}) do |row, memo|
```

Comment:
I'd appreciate it if for `each_with_objects` that return the result to have a more descriptive name than `memo`.

i.e. This `memo` is the `ancestor_id_tree`. In `pluck_ancestor_tree`, I'm not sure what you'd call it, but a name might help.